charts/pwa/ci/cache-channels.yaml

```diff
@@ -1,3 +1,4 @@
+# yaml-language-server: $schema=../values.schema.json
```

Comment:
I wasn't aware of that possibility..
To document this finding -> [yaml-language-server](https://github.com/redhat-developer/vscode-yaml#associating-a-schema-in-the-yaml-file)